src/rp2040_arduino_platform.h

```diff
@@ -13,11 +23,36 @@ class RP2040ArduinoPlatform : public ArduinoPlatform
     void setupUart();
 
     // unique serial number
-    uint32_t uniqueSerialNumber() override;
+    uint32_t uniqueSerialNumber(); //override; 
```

Comment:
Why was the override removed?

Reply:
when i started with the flash branch master was not merged into flash branch and uniqueSerialNumber not defined (so the override caused a compile error).
override is optional anyway, so i removed it. It can be added again, no problem.

Reply:
Ok. Please do so.

Reply:
done